packages/@instana-integration/python/README.md

```diff
@@ -0,0 +1,3 @@
+# @instana-integration/python
+
+The integration package is used to support Python monitoring. Once you import this package into your Instana environment, you will be able to monitor Python runtime and the applications on various aspects by checking the dashboards, alerts, etc. included in this integration package.
```

Comment:
The README file content may be updated by taking nodejs/go package as reference.

Reply:
Updated